console/console-init/ui/src/modules/device/components/DeviceFilter/DeviceFilter.tsx

```diff
@@ -96,11 +95,12 @@ const DeviceFilter: React.FunctionComponent<IDeviceFilterProps> = (
     // setFilterrere
   }
 ) => {
-  const [filter, setFilter] = useState<IDeviceFilter>(setInitialFilter());
+  const [filter, setFilter] = useState<IDeviceFilter>(getInitialFilter());
   const [lastAppliedFilter, setLastAppliedFilter] = useState<IDeviceFilter[]>([
-    setInitialFilter()
+    getInitialFilter()
   ]);
   const [isKebabOpen, setIsKebabOpen] = useState<boolean>(false);
+  const [isRedoEnabled, setIsRedoEnabled] = useState<boolean>(false);
   const typeOptions: ISelectOption[] = [
```

Comment:
can we move all dropdown options in constant file in same module. I would like to suggest create one constant file in utils at same module.